walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +315,162 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void everyRequiredAttendeeIsConsidered() {
```

Comment:
Just a thought: maybe a more intuitive name would be `optionalAttendeeIsNotConsidered` or `optionalAttendeeCannotAttend`

Reply:
Ah `optionalAttendeeCannotAttend` sounds pretty good, I'll change the name that.